api/backend/tests/test_ingestion_repository.py

```
@@ -0,0 +1,128 @@
+            "is_building_link": True,
+            "name": "Entry",
+        }
+    ]
+
+    def _fake_builder(osm_file_path: str) -> Tuple[list, list]:
+        captured_paths.append(osm_file_path)
+        return nodes_payload, edges_payload
+
+    monkeypatch.setattr(ingestion_repo, "build_access_graph", _fake_builder)
+
+    repo.populate_access_graph(city_id=1, file_path="/tmp/demo.pbf")
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp/demo.pbf"

(S108)